mat/symband.go

```diff
@@ -27,6 +28,22 @@ type SymBandDense struct {
 	mat blas64.SymmetricBand
 }
 
+// SymBanded is a symmetric band matrix interface type.
+type SymBanded interface {
```

Comment:
Wouldn't this be better expressed as `Banded` and then the extra `Symmetric`?

Reply:
(in terms of the expression of the interface methods)

Reply:
I couldn't decide which way that should go, so I avoided it. I guess this way is the lesser number of methods literally present. I'll change it.

Reply:
Though it's at odds with how the mutable type is expressed.

Reply:
Is it? MutableBanded is Banded plus a Mutable method

Reply:
`MuatbleSymBanded` is `Symmetric`+some methods. This is `Banded`+some methods.